fl4health/clients/nnunet_client.py

```diff
@@ -203,6 +207,58 @@ def __init__(
                 log(INFO, "Switching pytorch model jit compile to OFF")
             os.environ["nnUNet_compile"] = str("false")
 
+    def train_step(self, input: TorchInputType, target: TorchTargetType) -> Tuple[TrainingLosses, TorchPredType]:
```

Comment:
I'm not super familiar with mixed precision training so I'm going to trust that this does what its supposed to do.